bin/mocha

```diff
@@ -151,5 +151,5 @@ if (Object.keys(nodeArgs).length) {
     proc.kill('SIGTERM'); // if that didn't work, we're probably in an infinite loop, so make it die.
   });
 } else {
-  require('../lib/cli/cli').main(unparse(mochaArgs, {alias: aliases}));
+  require('../lib/cli/cli').main(['preParsedOptions', mochaArgs]);
```

Comment:
this feels like a pretty strange way to get this done.  I'd prefer a more "designed" approach to this, and one that can be tested.

at this point, it feels like premature optimization.  the overhead is *forking the child process*, which this PR avoids.  avoiding double-parsing will be much less of a perf gain.  either way, it can be addressed in another PR...  how would you feel about reverting this and creating another PR for it?

Reply:
@boneskull  It's ok, I will revert it.

@plroebuck which is `npx`s solution? This code is not self-explanatory for me. I can't imagine an alternative way without spawning. Before launching Node, we don't know wether there are Node options or not.